src/main/java/Duke.java

```diff
@@ -1,10 +1,291 @@
+
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+import src.main.java.Task;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+
+    private static ArrayList<String> stringList = new ArrayList<>();
+    private static ArrayList<Task> taskList = new ArrayList<>();
+    private static ArrayList<String> dueDate = new ArrayList<>();
+
+    private static int listCount = 0;
+
+    private static final String DATA_FILE_PATH = "./data/data.txt";
+
+    private static final String LINE = "-------------------------------------------------------------------\n";
+    private static final String GREET_USER = "Hello! I'm Duke \nWhat can I do for you?";
+    private static final String EXIT_MESSAGE = "Bye. Hope to see you again soon!";
+
+    private static void greetUser() {
+        System.out.println(GREET_USER + "\n \n" + LINE);
+    }
+
+    private static void markAsDone(String line) {
+        String taskNumber = line.substring(5);
+        int taskListElement = Integer.parseInt(taskNumber) - 1;
+        taskList.get(taskListElement).markAsDone();
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println("[X] " + taskList.get(taskListElement).getDescription());
+        System.out.println(LINE);
+    }
+
+    private static void list() {
+        int taskCount = 1;
+        for (int i = 0; i < listCount; i++) {
+            System.out.println(taskCount + "." + taskList.get(i));
+            taskCount++;
+        }
+        System.out.println(LINE);
+    }
+
+    private static Todo todo(String line) {
```

Comment:
Perhaps `todo`, `deadline`, and `event` could have better names, [especially as verbs, following the coding standards](https://se-education.org/guides/conventions/java/basic.html#naming)?